src/ng2-auto-complete.component.ts

```diff
@@ -138,10 +140,8 @@ export class Ng2AutoCompleteComponent implements OnInit {
   /**
    * constructor
    */
-  constructor(
-    elementRef: ElementRef,
-    public autoComplete: Ng2AutoComplete
-  ) {
+  constructor(elementRef: ElementRef,
+              public autoComplete: Ng2AutoComplete) {
```

Comment:
Let's not change the style unless it is an industry standard or community recommended